css-view-transitions-1/Overview.bs

```diff
@@ -1368,10 +1550,32 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 			Issue: Specify what happens with 'mix-blend-mode'.
 	</div>
 
-## [=Update transition DOM=] ## {#update-transition-dom-algorithm}
+### [=Compute the interest rectangle=] ### {#compute-the-interest-rectangle-algorithm}
+
+	<div algorithm>
+		To <dfn lt="computing the interest rectangle|compute the interest rectangle">compute the interest rectangle</dfn> of an {{Element}} |element|, perform the following steps.
+		They return a rectangle.
+
+		Note: The interest rectangle is the subset of |element|'s [=ink overflow rectangle=] that should be captured.
+			This is required for cases where an element's ink overflow rectangle needs to be clipped because of hardware constraints.
+			For example, if it exceeds the maximum texture size.
+
+		1. Assert: |element| is not |element|'s [=node document=]'s [=document element=].
+
+			Note: The [=document element=] is captured differently, as specified in [=capture the image=].
+
+		1. If |element|'s [=ink overflow area=] does not exceed an implementation-defined maximum size,
+			then return a rectangle that is equal to |element|'s [=ink overflow rectangle=].
+
+		1. Otherwise:
+
+			Issue: Define the algorithm used to clip the snapshot when it exceeds max size.
+	</div>
+
+## [=Handle transition frame=] ## {#update-transition-dom-algorithm}
```

Comment:
Renamed this. It used to be "update transition DOM", but it's doing more than that.

Reply:
Sounds good.